### ISSUE TITLE
Do not overwrite symbol names with autocomments

### ISSUE THREAD

**Issue by Mattiwatti** [x64dbg/x64dbg#1775]
Debugger version: any recent
OS/bitness: any

The register content autocomments overwrite symbol names. Example:
![comment](https://i.imgur.com/bHmXtk0.png)

The marked comment was initially the name of the function, `BaseSetLastNTError`.

I know you can disable this behaviour with the 'autocomments only on CIP' option, but I do appreciate the autocomments most of the time. Here though I would definitely prefer to see the function name (perhaps even in bold or black for emphasis) rather than the contents of `edi`.

Manual comments are not affected by this.

**Comment by mrexodia** [comment]
Yeah the symbols are displayed in the address column, you can expand it. This behavior is only to help out people who didn’t notice that 😀

**Comment by Mattiwatti** [comment]
D'oh! Now I feel stupid. I've always had the address column set to the exact minimum width required to contain an address.

I'd still prefer the autocomment not overwrite the symbol name (or otherwise be appended to it); with the address column expanded it feels like a lot of screen real estate is being wasted because most addresses do not have associated symbol names.

**Comment by mrexodia** [comment]
Yeah I agree that an option could be done for this. Also what do you think
about displaying a tooltip when mousing over the address column? I’m not
shre how I feel about that...

**Comment by Mattiwatti** [comment]
Hm, I dunno about tooltips... having them is definitely an improvement over no tooltips, but the address column isn't the first place I'd look for something like this. Of course, after you've seen the tooltip once you'll know where it is so maybe it's not such a big deal. Maybe it could be done in combination with the same mini-arrows that already exist for jump instructions (but pointing sideways)?

`761E79F2 >`

That way you still have the breadcrumb hint for more info but without the address bar taking up 2-3 times the normal width. It also prevents you from hovering over addresses needlessly because you can immediately see whether or not there will be a tooltip for it.

**Comment by torusrxxx** [comment]
We can disable autocomments for this particular instruction “mov edi,
edi”.It appears in system modules frequently and doesn’t have any meaning.

**Comment by Mattiwatti** [comment]
Offtopic/pointless comment, but maybe there is someone out there thinking "wait, why do these functions do `mov edi,edi` in the first place?" The answer is: they are padding bytes in order to support hotpatching. Microsoft invested millions and tons of developer time making almost every function in Windows (including the kernel) hotpatchable around... Windows XP/2003(?) times. So in theory you shouldn't *have* to reboot every time Windows Update updates flash player or some stupid DLL that's used by IE. But the grand total of updates that has actually made use of this functionality in reality is about 3.

**Comment by mrexodia** [comment]
The function label now seems to take priority over the autocomments.
